packages/map/src/map/utils.jsx

```diff
@@ -42,15 +48,32 @@ export const getMapOptions = ({
     attributionControl: !noAttribution,
     interactive,
 
+    bounds: getBoundingBox(bounds),
+
     scrollZoom: false,
     dragRotate: false,
+    dragPan: {
+      // Disable intertia on drag
+      maxSpeed: 0,
+    },
     pitchWithRotate: false,
+    fitBoundsOptions: { padding: fitPadding },
   };
 
-  if (defaultView) options.center = defaultView;
-  if (defaultZoom) options.zoom = defaultZoom;
-  if (isFilledArray(bounds)) options.bounds = getBoundingBox(bounds);
-  if (fitPadding) options.fitBoundsOptions = { padding: fitPadding };
+  if (isOnePoint(options.bounds)) {
```

Comment:
still allow to set min/max zoom. only set default maxZoom when no zoom prop was provided by the user